packages/mdc-checkbox/_mixins.scss

```diff
@@ -279,26 +451,51 @@
 
 // Focus indicator
 
-@mixin mdc-checkbox__focus-indicator_ {
-  @include mdc-checkbox__child--cover-parent_;
+@mixin mdc-checkbox__focus-indicator_($query: mdc-feature-all()) {
+  $feat-animation: mdc-feature-create-target($query, animation);
+  $feat-structure: mdc-feature-create-target($query, structure);
 
-  width: 100%;
-  height: 100%;
-  transform: scale(0, 0);
-  transition: mdc-checkbox-transition-exit(opacity), mdc-checkbox-transition-exit(transform);
-  border-radius: 50%;
-  opacity: 0;
-  pointer-events: none;
-  content: "";
-  will-change: opacity, transform;
+  @include mdc-feature-targets($feat-structure) {
```

Comment:
It seems safe to reorder the structure and animation blocks here to combine them.